src/main/zlatopolsky/chapter11/Ch1N11_158.java

```diff
@@ -0,0 +1,49 @@
+package zlatopolsky.chapter11;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class Ch1N11_158 {
+
+    public int[] removeDuplicate(int[] nums) {
+        nums = Arrays.stream(nums).distinct().toArray();
```

Comment:
// I think you need to learn about HashSet

Reply:
yes, i agree

Reply:
There is no need to assign result to "nums". Just return the value

Reply:
And also try to avoid reassigning to input parameters